### ISSUE TITLE
Make sending typing notifications optional

### ISSUE THREAD

**Issue by KitsuneRal** [quotient-im/libQuotient#382]

**Comment by redstrate** [comment]
IIUC this is already possible at least in modern libQuotient, right? The library consumer is in full control of whether `SetTypingJob` is used.

**Comment by KitsuneRal** [comment]
Indeed; I'm not even sure why I created this because neither libQuotient nor even Quaternion actively send typing notifications so far.
